docs/CodeOverview.md

```diff
@@ -0,0 +1,21 @@
+# Code OverView
+
+## EntryPoint and Folder Structure
+
+TODO: add description of entry point and repo structure
+
+## Code Style
+
+TODO: are we using some code style?
```

Comment:
Yes agree with move it to "Contributing" doc. I raised this because saw different coding style here and was thinking maybe can introduce a coding style and let apply later